server.go

```diff
@@ -209,7 +209,7 @@ func main() {
 	}
 	flag.StringVar(&flags.ConfigFile, "config", "", "Configuration filename")
 	flag.StringVar(&flags.MetricsAddr, "metrics", "", "Address for the Prometheus metrics")
-	flag.StringVar(&flags.IPCountryDB, "ip_country_db", "", "Path to the GeoLite2-Country.mmdb file")
+	flag.StringVar(&flags.IPCountryDB, "ip_country_db", "", "Path to the dbip-country-lite.mmdb file")
```

Comment:
There's nothing DB-IP-specific logic here. If the file name is different, we should still refer to the format. For example "GeoLite2-Country .mmdb file"

Reply:
There is, the schema of the database is specific to DB-IP, and this specifies the filename which is useful to have

Reply:
Still, we allow any MMDB file. This code must make no reference to DB IP. It's a generic SS server anyone can use with whatever file they want

Reply:
It doesn't allow any MMDB file, we only support databases with schemas supported by geoip2-golang

Reply:
I think it's still a good idea to have a generic name here. Maybe `Path to the ip-to-country.mmdb file`

Reply:
Done